src/Catalyst.Core.Modules.Ledger.Tests/IntegrationTests/LedgerKvmTests.cs

```diff
@@ -0,0 +1,332 @@
+﻿#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.Linq;
+using System.Reactive.Linq;
+using Catalyst.Abstractions.Consensus.Deltas;
+using Catalyst.Abstractions.Cryptography;
+using Catalyst.Abstractions.Hashing;
+using Catalyst.Abstractions.Kvm;
+using Catalyst.Abstractions.Mempool;
+using Catalyst.Core.Lib.DAO;
+using Catalyst.Core.Modules.Cryptography.BulletProofs;
+using Catalyst.Core.Modules.Hashing;
+using Catalyst.Core.Modules.Kvm;
+using Catalyst.Core.Modules.Ledger.Repository;
+using Catalyst.Protocol.Deltas;
+using Catalyst.TestUtils;
+using FluentAssertions;
+using Google.Protobuf.WellKnownTypes;
+using LibP2P;
+using Microsoft.Reactive.Testing;
+using Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Hosting;
+using Nethermind.Core;
+using Nethermind.Core.Extensions;
+using Nethermind.Core.Specs;
+using Nethermind.Dirichlet.Numerics;
+using Nethermind.Evm;
+using Nethermind.Evm.Tracing;
+using Nethermind.Logging;
+using Nethermind.Store;
+using NSubstitute;
+using TheDotNetLeague.MultiFormats.MultiHash;
+using Xunit;
+using ILogger = Serilog.ILogger;
+
+namespace Catalyst.Core.Modules.Ledger.Tests.IntegrationTests
+{
+    public class LedgerKvmTests
+    {
+        private Ledger _ledger;
+        private readonly IKvm _kvm;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_kvm' and declare it as a local variable in the relevant methods.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4350015)